### ISSUE TITLE
cannot find module error

### ISSUE THREAD

**Issue by arunimaram** [go-ldap/ldap#242]
Using `"github.com/go-ldap/ldap"` in import, go build spits out 
go: downloading gopkg.in/asn1-ber.v1 v1.3.1
`build trying: cannot load gopkg.in/asn1-ber.v1: cannot find module providing package gopkg.in/asn1-ber.v1`

Importing instead with `gopkg.in/ldap.v3`, fails with 
```
go: downloading gopkg.in/ldap.v3 v3.1.1
build trying: cannot load gopkg.in/ldap.v3: cannot find module providing package gopkg.in/ldap.v3
```

Basically gopkg seems to be failing in either case.

**Comment by johnweldon** [comment]
see #243

**Comment by johnweldon** [comment]
This should be fixed by the latest release v3.1.2
